api/graphql/makeModels.js

```diff
@@ -144,6 +144,7 @@ export default function makeModels (userId, isAdmin) {
         'created_at'
       ],
       relations: [
+        {post: {alias: 'post'}},
```

Comment:
I wasn't sure if it would work in the singular! All the other relations were 'tablename-s' :)